samples/bluetooth/peripheral_hids_mouse/Kconfig

```diff
@@ -16,7 +16,8 @@ config BT_HIDS_SECURITY_ENABLED
 	select FLASH
 	select FLASH_PAGE_LAYOUT
 	select FLASH_MAP
-	select NVS
+	select ZMS if SOC_FLASH_NRF_RRAM
+	select NVS if !SOC_FLASH_NRF_RRAM
```

Comment:
Should be imply so it can be changed to allow for relocating to/from external storage